materialsystem/shaderapidx9/shaderdevicedx8.cpp

```diff
@@ -1870,7 +1870,7 @@ void CShaderDeviceDx8::ShutdownDevice()
 //-----------------------------------------------------------------------------
 bool CShaderDeviceDx8::IsUsingGraphics() const
 {
-	//*****LOCK_SHADERAPI();
+	//LOCK_SHADERAPI();

```

Comment:
Why remove the asterisks here?

Reply:
Sorry, caused the entire paged to seem commented in my editor. Will revert.

Reply:
Can you replace it with exclamation marks then?

Reply:
Fixed.